.github/workflows/maven.yml

```diff
@@ -8,32 +7,22 @@ on:
     branches: [ "master" ]
 
 jobs:
-  java-8:
-
+  build:
     runs-on: ubuntu-latest
 
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up JDK 8
-        uses: actions/setup-java@v3
-        with:
-          java-version: '8'
-          distribution: 'adopt'
-          cache: maven
-      - name: Build with Maven
-        run: mvn clean install --file pom.xml
-
-  java-11:
-
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11, 21]
```

Comment:
Just clarifying, does this mean we are still testing against Java 8, I thought part of this ticket was dropping support for versions of java under 21?

Reply:
On this module, I was able to support both Java 8 and 21 without getting any errors. @dkayiwa/ @ibacher, should we drop it?

Reply:
I'm fine with still supporting Java 8 if there is no downside, makes it easiest for us!

Reply:
I was also under the impression that we'd be changing things to Java 21 format, but I suppose as long as it compiles with JDK 21 and runs with JDK 21, that's all we really need. And that might allow us to take a more piecemeal way of updating modules, which is always a good thing.